replace_app_name.py

```
@@ -0,0 +1,98 @@
+        full_path = project_root / file_path
+        if full_path.exists():
+            update_file(full_path, old_name, new_name)
+    
+    print(f"✅ Successfully replaced '{old_name}' with '{new_name}'")
+    print(f"📝 Don't forget to update your INSTALLED_APPS and URL imports!")
+
+
+def update_file(file_path: Path, old_name: str, new_name: str):
+    """Update a single file."""
+    try:
+        with open(file_path, 'r', encoding='utf-8') as f:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)